### PR TITLE
store: support time monitor.

### PR DESCRIPTION
**Pull request by siddontang**
Like TiDB time monitor. 
@ngaut @BusyJay @disksing @zhangjinpeng1987 @overvenus

### CODE REVIEW

**Comment by ngaut** [comment]
Any tests?

**Comment by siddontang** [comment]
No way to test, how to change system in test? @BusyJay

**Comment by BusyJay** [comment]
I don't think we should change system time. Mocking the system time is the common way to test it.

**Comment by ngaut** [comment]
Please see the implementation of TiDB.

**Comment by siddontang** [comment]
I don't think we need mock test here,  we should define a trait and then pass to store, needless.

**Comment by BusyJay** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -594,6 +595,7 @@ fn main() {
             run_local_server(listener, &cfg);
         }
         RAFTKV_DSN => {
+            let _ = TimeMonitor::default();
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: I think the monitor will be dropped immediately.

**Comment by siddontang** [comment]
PTAL

**Comment by BusyJay** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**